charts/gardener/controlplane/charts/application/templates/rbac-user.yaml

```diff
@@ -22,6 +22,22 @@ rules:
   - namespaces
   verbs:
   - get
+  - list
+  - watch
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - clusterroles
+  - clusterrolebindings
```

Comment:
What about `roles` and `rolebindings`?

Reply:
The are implicitly granted by the clusterrole with label `gardener.cloud/role: project-member`.
I will add them explicitly.